pep-0669.rst

```diff
@@ -289,26 +289,18 @@ Backwards Compatibility
 
 This PEP is mostly backwards compatible.
 
-This PEP is incompatible with :pep:`523` as the behavior would be undefined,
-since the VM has no control over the behavior of :pep:`523` plugins.
-
-Thus, if ``_PyInterpreterState_SetEvalFrameFunc()`` has been called and has
-changed the frame evaluation function, then calling
-``sys.monitoring.set_events()``, ``sys.monitoring.set_local_events()``, or
-``sys.monitoring.insert_marker`` will raise an exception.
-
-Likewise, if ``sys.monitoring.set_events()``,
-``sys.monitoring.set_local_events()``, or ``sys.monitoring.insert_marker``
-has been called, then calling ``_PyInterpreterState_SetEvalFrameFunc()``
-will raise an exception.
-
-``sys.settrace`` and ``sys.setprofile`` will act as if they were tools 6 and 7
-respectively, so can be used along side this PEP.
-
-This makes ``sys.settrace`` and ``sys.setprofile`` incompatible with :pep:`523`.
-Arguably, they already were as the author does not know of any PEP 523 plugin
-that supports ``sys.settrace`` or ``sys.setprofile`` correctly.
-This PEP merely formalizes that.
+There are some compatibility issues with :pep:`523`, as the behavior
+of :pep:`523` plugins is outside of the VM's control.
+It is up to :pep:`523` plugins to ensure that they respect the semantics
+of this PEP. Simple plugins that do not change the state of the VM, and
+defer execution to ``_PyEval_EvalFrameDefault()`` should continue to work.
+
+``sys.settrace`` and ``sys.setprofile`` will act as if they were tools
```

Comment:
Just FYI, you can actually link these with `` :func:`sys.settrace` ``, etc., if you want, just like in the CPython docs, since we added intersphinx support